### ISSUE TITLE
Title background

### ISSUE THREAD

**Issue by phaitour** [sberrevoets/SDCAlertView#74]
Is there way to add an image behind the title?  Or if that's not possible, set a solid color background behind the title?

**Comment by sberrevoets** [comment]
Yeah, I'm aware of that issue. There are 2 solutions to that. The first one is to fix SDCIntrinsicallySizedView to take the negative insets into account, and the second one is to create a height constraint on the `contentView` that forces it to be taller.

The first solution would be better, but I don't know how soon I can get to that. If you want to do it and submit a PR, I'd appreciate it. Alternatively, you can opt for the second solution in your project which will fix it as well.

Your call :smiley:

**Comment by phaitour** [comment]
Thanks!  I played around with `contentPadding` inside the visual style and I was able to get rid of the space above the `contentView`.  With that gone, this entire thing was a none issue.

Thanks again for your help!

**Comment by sberrevoets** [comment]
Ah yes, forgot about that one. Glad you found a way!

**Comment by phaitour** [comment]
There's a small bug in the legacy alert's autolayout constraints.  It assumes a title/message will always exist and thus injects a blank space above the `contentView` even when the title and message are both missing.  I'll send you a PR.
